import/popups/ShareLinkPage.qml

```diff
@@ -1,32 +0,0 @@
-/****************************************************************************
-**
-** Copyright (C) 2013 Jolla Ltd.
-** Contact: Antti Seppälä <antti.seppala@jollamobile.com>
-**
-****************************************************************************/
-
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this file,
- * You can obtain one at http://mozilla.org/MPL/2.0/. */
-
-import QtQuick 2.0
-import Sailfish.Silica 1.0
-import Sailfish.TransferEngine 1.0
-
-SharePage {
-    id: page
-
-    property string link
-    property string linkTitle
-
-    //: List header for link sharing method list
-    //% "Share link"
-    header: qsTrId("sailfish_components_webview_popups-he-share_link")
```

Comment:
unlikely, but just checking: did something else use this translation ID without specifying the EE translation and context?

Reply:
Not within this repo, at least.